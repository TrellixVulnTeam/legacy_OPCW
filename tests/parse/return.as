# A function that has an explicit return statement returning nothing.
def main() { return; }

# A function that explicitly returns something.
def some(): int { return 4; }
