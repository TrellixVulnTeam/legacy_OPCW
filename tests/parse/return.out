ModuleDecl <?> _
  Nodes <?> 
    FuncDecl <?> main
      Parameters <?> 
      Nodes <?> 
        ReturnExpr <?> 
    FuncDecl <?> some 'int'
      Parameters <?> 
      Nodes <?> 
        ReturnExpr <?> 
          IntegerExpr <?> 'int' 4 (10)
